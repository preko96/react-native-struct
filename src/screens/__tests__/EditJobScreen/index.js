import React from 'react'
import { EditExistingJobScreen } from '../../EditJobScreen/'

let wrapper
const mockProps = {
	jobID: 'jobID',
	Company: 'Company',
	SubBuilding: 'SubBuilding',
	BuildingName: 'BuildingName',
	BuildingNumber: 'BuildingNumber',
	PrimaryStreet: 'PrimaryStreet',
	SecondaryStreet: 'SecondaryStreet',
	Town: 'Town',
	County: 'County',
	Postcode:'Postcode',
	ManualAddressReason: 'ManualAddressReason',
	activeJob: 'activeJob',
}

beforeEach(() => {
	wrapper = shallow(<EditExistingJobScreen {...mockProps}/>)
})

it('renders without crashing', () => {
	expect(wrapper).toMatchSnapshot()
})