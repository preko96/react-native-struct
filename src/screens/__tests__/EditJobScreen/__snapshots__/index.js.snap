// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Styled(Container)>
  <Styled(Header)>
    <Styled(Left)>
      <Connect(BackButton) />
    </Styled(Left)>
    <Styled(Body)>
      <Styled(Text)>
        Edit Job
      </Styled(Text)>
    </Styled(Body)>
    <Styled(Right)>
      <Connect(HeaderRightButton)
        editedJob={
          Object {
            "BuildingName": "BuildingName",
            "BuildingNumber": "BuildingNumber",
            "Company": "Company",
            "County": "County",
            "ManualAddress": undefined,
            "ManualAddressReason": "ManualAddressReason",
            "Postcode": "Postcode",
            "PrimaryStreet": "PrimaryStreet",
            "SecondaryStreet": "SecondaryStreet",
            "SubBuilding": "SubBuilding",
            "Town": "Town",
            "activeJob": "activeJob",
            "jobID": "jobID",
          }
        }
      />
    </Styled(Right)>
  </Styled(Header)>
  <Styled(Content)>
    <StyledInput
      onChangeText={[Function]}
      placeholder="Company"
      value="Company"
    />
    <StyledInput
      onChangeText={[Function]}
      placeholder="Sub Building"
      value="SubBuilding"
    />
    <StyledInput
      onChangeText={[Function]}
      placeholder="Building Name"
      value="BuildingName"
    />
    <StyledInput
      onChangeText={[Function]}
      placeholder="Building Number"
      value="BuildingNumber"
    />
    <StyledInput
      onChangeText={[Function]}
      placeholder="Primary Street (Required)"
      value="PrimaryStreet"
    />
    <StyledInput
      onChangeText={[Function]}
      placeholder="Secondary Street"
      value="SecondaryStreet"
    />
    <StyledInput
      onChangeText={[Function]}
      placeholder="Town"
      value="Town"
    />
    <StyledInput
      onChangeText={[Function]}
      placeholder="County"
      value="County"
    />
    <StyledInput
      onChangeText={[Function]}
      placeholder="Postcode (Required)"
      value="Postcode"
    />
  </Styled(Content)>
</Styled(Container)>
`;
